pkg/controllers/vdb/tlsservercertgen_reconciler.go

```diff
@@ -57,12 +58,14 @@ func MakeTLSServerCertGenReconciler(vdbrecon *VerticaDBReconciler, log logr.Logg
 // Reconcile will create a TLS secret for the http server if one is missing
 func (h *TLSServerCertGenReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ctrl.Result, error) {
 	secretFieldNameMap := map[string]string{
-		nmaTLSSecret: h.Vdb.Spec.NMATLSSecret,
+		nmaTLSSecret:          h.Vdb.Spec.NMATLSSecret,
+		clientServerTLSSecret: h.Vdb.Spec.ClientServerTLSSecret,
 	}
 	err := error(nil)
 	for secretFieldName, secretName := range secretFieldNameMap {
 		err = h.reconcileOneSecret(secretFieldName, secretName, ctx)
 		if err != nil {
+			h.Log.Error(err, "failed to reconcile secret for "+secretFieldName)
```

Comment:
```suggestion
			h.Log.Error(err, "failed to reconcile secret for %s", secretFieldName)
```
Please use this format for logging.

Reply:
structured logging is hard to read. I prefer one line format.